website/project/model.py

```diff
@@ -658,8 +658,16 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     }
 
     def __init__(self, *args, **kwargs):
+        tags = kwargs.get('tags')
+        if tags:
+            del kwargs['tags']
+
         super(Node, self).__init__(*args, **kwargs)
 
+        if tags:
```

Comment:
Discussed in person: Any special-casing should be accompanied with a comment explaining why it's necessary.